ddtelemetry/src/worker/mod.rs

```diff
@@ -601,8 +607,8 @@ impl TelemetryWorker {
                 }
             }
             AppHeartbeat(()) | AppClosing(()) => {}
-            // TODO Paul lgdc keep metrics until we know if the flush was a success
-            GenerateMetrics(_) | Distributions(_) => {}
+            // TODO: Paul lgdc keep metrics until we know if the flush was a success
```

Comment:
Should we add a ticket for this in the backlog? Is this a real issue? Aren't we ok to drop one payload (as long as we log, at least in debug, that we drop them)

Reply:
@paullegranddc ?

Reply:
i think's it's fine to lose metrics payloads from time to time